db/fetcher/fetcher.go

```diff
@@ -57,6 +60,7 @@ type Fetcher interface {
 	Init(
 		ctx context.Context,
 		txn datastore.Txn,
+		acp immutable.Option[acp.ACPModule],
```

Comment:
todo: This is already on the collection no?  Please remove the new parameter.  The same goes for the private `init` func, and `acp` `DocumentFetcher` property.  And the index, versioned and lens fetchers.

Reply:
I got mixed up here and confused it with the policy :)